Base/QTGUI/qSlicerSaveDataDialog.cxx

```diff
@@ -965,7 +965,7 @@ vtkMRMLNode* qSlicerSaveDataDialogPrivate::getNodeByID(char *id, vtkMRMLScene* s
       {
       vtkMRMLSceneViewNode *svNode = vtkMRMLSceneViewNode::SafeDownCast(*it);
       // skip "Master Scene View" since it contains the same nodes as the scene
-      if (svNode->GetName() && std::string("Master Scene View") == std::string(svNode->GetName()))
+      if (svNode->GetName() && std::string(/*no tr*/"Master Scene View") == std::string(svNode->GetName()))
```

Comment:
Search the full Slicer code base and mark all usages of "Master Scene View" as "no tr", even in VTK classes.